src/package-manifest.ts

```diff
@@ -139,7 +139,10 @@ export function readPackageManifestNameField(
 function isValidPackageManifestVersionField(
```

Comment:
I see that this function is called `isValidPackageManifestVersionField`, but it seems that we are using this for not only checking that the "version" field of a `package.json` is correct, but also that the version for a dependency is correct. A package's version can't be `workspace:^`, so perhaps we want to leave this function unchanged and then copy it to a new function called `isValidPackageManifestDependencyValue` where we can make this change? Then we can have `isValidPackageManifestDependenciesField` use this new function instead.

Reply:
Yes indeed didn't pay attention to I removed the semver valid range check for the initial function as it was not there it was added later in this commit https://github.com/MetaMask/create-release-branch/pull/106

Reply:
Oh that's okay! I think it makes a lot of sense to keep the `isValidSemver` check in `isValidPackageManifestVersionField`. No reason to change that. It's just that the `workspace:^` check doesn't apply there, it only applies to dependency versions.